Cannot find name 'CustomMatcherResult'

Getting the following error when trying to build a project that references v0.5.0

```
node_modules/jest-file-snapshot/index.d.ts:9:6 - error TS2304: Cannot find name 'CustomMatcherResult'.

9 ) => CustomMatcherResult;
       ~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -6,7 +6,7 @@ export const toMatchFile: (
   content: string | Buffer,
   filename?: string,
   options?: FileMatcherOptions
-) => CustomMatcherResult;
+) => jest.CustomMatcherResult;
 
 declare interface FileMatcherOptions {
   diff?: DiffOptions;

